profiles/test/data/VP_LUNARG_test_api.json

```diff
@@ -2983,13 +2983,26 @@
                             "depth": 381
                         }
                     },
-                    "VkQueueFamilyGlobalPriorityPropertiesEXT": {
+                    "VkQueueFamilyProperties2": {
+                        "queueFamilyProperties": {
+                            "queueFlags": [ "VK_QUEUE_COMPUTE_BIT", "VK_QUEUE_TRANSFER_BIT" ],
+                            "queueCount": 382,
+                            "timestampValidBits": 383,
+                            "minImageTransferGranularity": {
+                                "width": 384,
+                                "height": 385,
+                                "depth": 386
+                            }
```

Comment:
You should also add some tests in which not all fields are defined, e.g. `VkQueueFamilyProperties` without `timestampValidBits`, or something, just to make sure that parsing works, even when partial structure data are provided.

I guess in such cases the `timestampValidBits` should actually come from the real device's queue family properties, and just the properties defined by the profile should be overridden, but I'm not sure this was thought through before, so maybe we can just default it to zero.

Nonetheless, the layer should be able to handle in some way when some fields aren't defined by the profile.